system/include/emscripten/asmfs.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// Prints the contents of the whole ASMFS filesystem to console
+void emscripten_asmfs_dump(void);
```

Comment:
I assume this is just for debugging (if so maybe a comment about that makes sense)? Or is there another use?

Reply:
That's true - added a comment.